app/presenters/detailed_guide_presenter.rb

```diff
@@ -43,6 +43,24 @@ def related_mainstream
     links("related_mainstream")
   end
 
+  def other
+    hash = {}
+    if !applies_to_all_nations
+      hash["Applies to"] = applies_to
```

Comment:
`government-frontend` has no translations. I think we should actually send in the translated keys in the schema, e.g.:

```diff
    "applicable_nations": {
      "all_nations": false,
-     "nations": ["england"]
+     "nations": ["England"]
    }
```